app/src/main/java/com/abhijitvalluri/android/fitnotifications/services/NLService.java

```diff
@@ -129,7 +134,7 @@ public static void onAppSelectionsUpdated(Context context) {
 
     public static void onPlaceholderNotifSettingUpdated(boolean dismissNotif, int delaySeconds) {
         mDismissPlaceholderNotif = dismissNotif;
-        mPlaceholderNotifDismissDelayMillis = delaySeconds*1000;
+        mPlaceholderNotifDismissDelayMillis = delaySeconds * 1000;
```

Comment:
It is a "common wisdom" in code style to put spaces around bitwise operators for better code readability. You actually do the same yourself when concatenating strings, for example.
I was just trying to make it consistent.

P.S. If you have specific code formatting requirements, please export them into the IDE project.